<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagementDialog</class>
 <widget class="QDialog" name="UserManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj nowego użytkownika:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddUser">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń użytkownika po ID:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputDeleteId">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteUser">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
  <widget class="QComboBox" name="inputRole">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>waiter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>manager</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>admin</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="inputPassword">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>75</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputUsername">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>75</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
